classes/Link.php

```diff
@@ -161,6 +161,10 @@ public function getProductLink(
             $params['id'] = $product->id;
         }
 
+        //Attribute equal to 0 is useless, so we set to null so that it won't be inserted in url parameters
+        if (0 === $ipa) {
```

Comment:
why not checking for empty, in case someone insert an empty string?

Reply:
Good idea!

Reply:
@PierreRambaud Done! Thanks